### PR TITLE
HotFix to Make Ligand Mol2 Bond Type

### PR DESCRIPTION
**Pull request by blakeo2**
- "Ligand" is not a valid bond type in the mol2 format. For now, it is being assigned as an "unknown" bond type

### CODE REVIEW

File: molli/chem/bond.py [hunk 1]
```diff
@@ -477,6 +477,10 @@ def get_mol2_type(self) -> str:
 
             case BondType.NotConnected:
                 return MOL2_BOND_TYPE_MAP.inverse[BondType.NotConnected]
+            
+            #Mol2 Format Doesn't Recognize Multi-Attachment Bonds Natively
+            case BondType.Ligand:
```
Comment (esalx) [inline molli/chem/bond.py:482]: Should we do `case _` here? This will match any bond type, seems like a good fit for mol2.

**Review by esalx — CHANGES_REQUESTED** [review]

File: molli/chem/bond.py (hunk 1 above)
Comment (blakeo2) [reply molli/chem/bond.py:482]: I'm fine with that, would you prefer it separate or remove the bondtype.ligand class?

Comment (esalx) [reply molli/chem/bond.py:482]: Let's do separate, it will be easier to maintain.

Comment (blakeo2) [reply molli/chem/bond.py:482]: Done!

**Review by esalx — APPROVED** [review]
Good work @blakeo2! Approved.

**Pull request merged**